site/gdocs/components/OwidGdocHeader.tsx

```diff
@@ -131,24 +120,19 @@ function OwidTopicPageHeader({
             <p className="topic-page-header__subtitle body-1-regular col-start-2 span-cols-8">
                 {content.subtitle}
             </p>
-            <p className="topic-page-header__byline col-start-2 span-cols-8 col-sm-start-2 span-sm-cols-12">
```

Comment:
Nitpick, but I think I'd have given `Byline` a `className` prop, make it render a `<p>` instead of a fragment, and handle `names.length === 0` in the component itself. I can see there's currently one place where it's being rendered inside a `<div>` but it's probably correct to have that be a `<p>` anyway.

Reply:
good call, thanks!